pkg/operator/ceph/csi/cluster_config.go

```diff
@@ -170,6 +170,9 @@ func updateCsiClusterConfig(curr, clusterKey string, newCsiClusterConfigEntry *C
 			// update default clusterID's entry
 			if clusterKey == centry.Namespace {
 				centry.Monitors = newCsiClusterConfigEntry.Monitors
+				centry.ReadAffinity = newCsiClusterConfigEntry.ReadAffinity
```

Comment:
What is the difference between the `for` loop on line 168 and the loop lower on line 180? It seems there is a lot of duplication between these two loops.